CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [v1.0.2] - 2018-07-23
+### Added
```

Comment:
I think this is just a fix not an addition because you fixed how the environment variable were populated. A "added" is not a patch anymore and you should increase the version to 1.1.0 if that is the case